tests/gui_tests/test_column_data.py

```diff
@@ -1374,6 +1557,8 @@ def test_edit_data_platform():
         },
         "privacy": {
             "foreign_table_type": TableTypes.REFERENCE,
+            "multiple_values_allowed": False,
```

Comment:
We need to lose one of these duplicated lines, please @robintw 

(though I expect it doesn't have runtime impact)

Reply:
Fixed now.